app/models/manageiq/providers/ibm_cloud/inventory/collector/power_virtual_servers.rb

```diff
@@ -38,4 +38,23 @@ def ports(network_id)
   def sshkeys
     connection.get_ssh_keys
   end
+
+  def system_pool
+    connection.get_system_pool
```

Comment:
@Kuldip-Nanda what does a `storage_pool` response look like?  Wondering if Flavor is the right way to model this or if we should do something else

Reply:
collector.system_pool.values returns json list of available system pool (Hardware platform detailing its limits and statistics) within a particular data center. I just wanted to adhere to the practice of not interpreting the return value. The parsing of the system pools object is giving the system available. I can change it to collector.system_pool.values

Reply:
You can return an array of system types (all that we're really collecting right now) with `connection.get_system_pool.keys`

Reply:
I am returning collector.system_pool.values as suggested by Adam.